src/routes/app/(flows)/funder-onboarding/+page.svelte

```diff
@@ -31,6 +31,7 @@
 <Stepper
   bind:currentStepIndex
   on:stepChange={() => window.scrollTo({ top: 0 })}
+  on:conclude={() => window.history.back()}
```

Comment:
what if you got linked to the flow from somewhere else, e.g. google or a discord link?